core/src/main/scala/org/apache/spark/status/storeTypes.scala

```diff
@@ -134,6 +138,7 @@ private[spark] object TaskIndexNames {
   final val STAGE = "stage"
   final val STATUS = "sta"
   final val TASK_INDEX = "idx"
+  final val COMPLETION_TIME = "completionTime"
```

Comment:
Could you use a shorter name like the others? It saves a little bit more space on disk because there are so many tasks in large apps.

Reply:
+1

Reply:
No, right now there's no support for that for indices.

Reply:
+1